tartufo/config.py

```diff
@@ -217,15 +216,19 @@ def load_rules_from_file(rules_file: TextIO) -> Dict[str, Rule]:
             rule = Rule(
                 name=rule_name,
                 pattern=re.compile(rule_definition["pattern"]),
-                path_pattern=re.compile(path_pattern) if path_pattern else None,
+                path_pattern=re.compile(path_pattern)
+                if path_pattern
+                else re.compile(""),
```

Comment:
Super minor nit: Can we move this up to a constant, `EMPTY_PATTERN = re.compile("")`? That way we don't have to potentially recompile an empty string regex many times. Even if it is a super low-cost operation, might as well minimize it.